portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,24 +28,40 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
-  private ArrayList<String> trial_array;
+public final class DataServlet extends HttpServlet {
+  private ArrayList<String> comments_array;
   private String json_array;
```

Comment:
Please change to camel case

https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names